vips/vip-541/bsctestnet.ts

```diff
@@ -0,0 +1,121 @@
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { ProposalMeta, ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+
+import { cutParams as params } from "./cut-params-testnet.json";
+
+const { bsctestnet } = NETWORK_ADDRESSES;
+
+export const vBNB_ADMIN = "0x04109575c1dbB4ac2e59e60c783800ea10441BBe";
+export const vBNB_ADMIN_IMPL = "0x920863fB3965fc411A1c0aC610C768F4347570fE";
+export const DEFAULT_PROXY_ADMIN = "0x7877ffd62649b6a1557b55d4c20fcbab17344c91";
+export const DIAMOND = "0x649616739bab52E2A98BC74d93c896Ca45944359";
+export const COMPTROLLER_LENS = "0x3ec96D6a9a14ee57aB83F81BB7386EBE515936D1";
+export const LIQUIDATOR = "0x55AEABa76ecf144031Ef64E222166eb28Cb4865F";
+export const LIQUIDATOR_IMPL = "0xe442A62E3B1956EC5B42e06aA0E293A0cB300406";
+export const LIQUIDATOR_PROXY_ADMIN = "0x1469AeB2768931f979a1c957692e32Aa802dd55a";
+export const VAI_CONTROLLER = "0xf70C3C6b749BbAb89C081737334E74C9aFD4BE16";
+export const VAI_CONTROLLER_IMPL = "0x5864e8BE0d4AD825feD65115a4f109f850A65aF7";
+export const VTOKEN_DELEGATE = "0x585C508aF088123d990182a19e655ebB0e540CA1";
+
+export const vTokens = {
+  vAAVE: "0x714db6c38A17883964B68a07d56cE331501d9eb6",
+  vADA: "0x37C28DE42bA3d22217995D146FC684B2326Ede64",
+  vasBNB: "0x73F506Aefd5e169D48Ea21A373B9B0a200E37585",
+  vBTC: "0xb6e9322C49FD75a367Fcb17B0Fcd62C5070EbCBe",
+  vBUSD: "0x08e0A5575De71037aE36AbfAfb516595fE68e5e4",
+  vCAKE: "0xeDaC03D29ff74b5fDc0CC936F6288312e1459BC6",
+  vDOGE: "0xF912d3001CAf6DC4ADD366A62Cc9115B4303c9A9",
+  vETH: "0x162D005F0Fff510E54958Cfc5CF32A3180A84aab",
+  vFDUSD: "0xF06e662a00796c122AaAE935EC4F0Be3F74f5636",
+  vlisUSD: "0x9447b1D4Bd192f25416B6aCc3B7f06be2f7D6309",
+  vLTC: "0xAfc13BC065ABeE838540823431055D2ea52eBA52",
+  vLUNA: "0x9C3015191d39cF1930F92EB7e7BCbd020bCA286a",
```

Comment:
Aren't LUNA and UST delisted? If not, I think we shouldn't upgrade them, but we should rather delist them